.github/workflows/Issue Templates/bug.yml

```diff
@@ -0,0 +1,44 @@
+name: Bug Report🐛
```

Comment:
Issue Templates should be in `.github/` not in `.github/workflows`, and for folder name again use camel case